lib/ui/retraced.ts

```diff
@@ -20,14 +20,14 @@ export const useProject = (projectId: string) => {
   };
 };
 
-export const useProjects = () => {
+export const useProjects = (paginate: any, pageLimit: number) => {
```

Comment:
I think it is better to accept `(offset: number, limit: number)` as parameters. This way it way more clear.